### ISSUE TITLE
pinpoint When to support dubbo2.7.x

### ISSUE THREAD

**Issue by ciweigg2** [naver/pinpoint#5617]

**Comment by RoySRose** [comment]
Hello, @ciweigg2 

Have you tried?
Somehow I get the feeling that Dubbo 2.7.x won't be supported, since the package name has been changed when Dubbo project moved to Apache foundation and few Classes has been deleted

Hello, @majinkai 
Do you think you can update the plugin and the Integration Test?

**Comment by ciweigg2** [comment]
I think update dubbo version and update packing this plugin is ok i try

**Comment by RoySRose** [comment]
What do you mean? I don't understand what you're trying to say.
Can you rephrase it?

**Comment by ciweigg2** [comment]
If I change the package name and version of dubbo, I change whether the original plug-in is available

**Comment by qq295190549** [comment]
Hi @ciweigg2 
I have developed a plugin for dubbo2.7.0.

**Comment by RoySRose** [comment]
@qq295190549 

Great!!!!
